src/Plugin/Plugin.php

```diff
@@ -86,7 +86,7 @@ public function isEmpty(): bool
      *
      * @psalm-return Generator<PluginVersionInterface>
```

Comment:
Should we change this to `@return Traversable<PluginVersionInterface>` as IDEs have now catched up with the generic notations?